package.json

```diff
@@ -59,8 +59,11 @@
     "start": "npm-run-all -p watch-css start-ts",
     "build-ts": "react-scripts-ts test -u --env=jsdom --coverage --bail --ci && react-scripts-ts build",
     "build": "npm-run-all build-css build-ts",
-    "test": "react-scripts-ts test -u --env=jsdom --coverage",
-    "test:watch": "react-scripts-ts test --env=jsdom --watch",
+    "test": "react-scripts-ts test --env=jsdom --coverage",
+    "test:watchAll:silent": "react-scripts-ts test --env=jsdom --watchAll --silent --verbose",
```

Comment:
I see here both the `--silent` and the `--verbose` flag for the silent task. Are both needed? The same for the test:watchChanged:silent.

Reply:
I explained the difference in visuals above. Combination of `silent` and `verbose` adds extra detail namely the descriptions of the tests run in nested structure which I find more descriptive than reviewing the number of tests run using only `silent`